www/src/models/entity/EntityStore.ts

```diff
@@ -1,12 +1,36 @@
 import { RStore } from "gena-app";
 import { SERVER } from "../../env";
 import { Entity } from "./Entity";
+import axios, { AxiosRequestConfig } from "axios";
+
+export interface EntityTextSearchResult {
+  id: string;
+  label: string;
+  description: string;
+  uri: string;
+}
 
 export class EntityStore extends RStore<string, Entity> {
   constructor() {
     super(`${SERVER}/api/entities`, undefined, false);
   }
 
+  /**
+   * Get search results from the search API from axios.
+   *
+   * @returns Promise<SearchResult[]> if there is no search result from Wikidata API.
+   */
+  async fetchSearchResults(
```

Comment:
nit picking here, but can you change the name of the function to findByName and the parameter to query? fetchSearchResults doesn't make lots of sense to me

Reply:
Updated the function name and comments.